display/ssd1306.h

```diff
@@ -121,15 +121,22 @@ class StandardDisplayController : public DisplayControllerBase<Width, col_t>, Sa
 	}
         return 200;  // redraw once every 200 ms
 
-      case SCREEN_MESSAGE:
+      case SCREEN_MESSAGE: {
 	Clear();
+     // Aurebesh Font option.
+      #ifdef USE_AUREBESH_FONT
+        const Glyph* font = Aurebesh10pt7bGlyphs;
+      #else
+        const Glyph* font = Starjedi10pt7bGlyphs;
+      #endif
         if (strchr(message_, '\n')) {
-          MonoFrame<WIDTH, col_t>::DrawText(message_, 0, 15, Starjedi10pt7bGlyphs);
+          MonoFrame<WIDTH, col_t>::DrawText(message_, 0, 15, font);
         } else {
 	  // centered
-          MonoFrame<WIDTH, col_t>::DrawText(message_, 0, HEIGHT / 2 + 7, Starjedi10pt7bGlyphs);
+          MonoFrame<WIDTH, col_t>::DrawText(message_, 0, HEIGHT / 2 + 7, font);
         }
         screen_ = SCREEN_PLI;
+      }
         return font_config.ProffieOSFontImageDuration;
```

Comment:
Ah ha! I thought that might be and that would explain why it seems that duration is being ignored in my other testing. In my previous PR when adding these braces to create a scope for the "font" variable so that it doesn't cross any "case" jump targets, you had said that the closing brace should go "somewhere before the break".
So thinking return and break are similar, that's why I had placed the brace before the return.

Fixed.

Reply:
No, it won't explain anything, as it doesn't change any behavior.
It's just a style thing.